tests/unit/test_intercept.py

```diff
@@ -30,6 +30,29 @@ def test_read_write_exec_trace(self):
             result = sut.parse_exec_trace(temp_file)
             self.assertEqual(input_one, result)
 
+    def test_read_write_exec_trace_with_response(self):
+        with libear.temporary_directory() as tmp_dir:
+            response_file_one = os.path.join(tmp_dir, 'response1.jom')
+            response_file_two = os.path.join(tmp_dir, 'response2.jom')
+            input_one = Execution(
+                pid=123,
+                cwd='/path/to/here',
+                cmd=['cc', '-c', '@'+response_file_one, 'this.c',
+                     '@'+response_file_two])
+            output_one = Execution(
+                pid=123,
+                cwd='/path/to/here',
+                cmd=['cc', '-c', 'that.c', 'this.c', '-Idoes_not_exists'])
```

Comment:
im not hugely into memorizing what compiler calls do work and what compiler .. but one can definitely compile multiple units  like that with cl/clang-cl - thats quite what the whole response file is usually doing: it containts multiple source entries .. 

But yeah, i made the test data to reflect your point of view so that users wont get confused.